### PR TITLE
Update the Getting Started tutorial.

### PR DESCRIPTION
**Pull request by wintermeyer**
A green field approach for this tutorial. Within 10 minutes a reader can create a fresh Phoenix Framework application with a working Ash authentication.

### CODE REVIEW

File: documentation/tutorials/getting-started-with-ash-authentication-phoenix.md [hunk 1]
```diff
@@ -1,87 +1,365 @@
...
-[
+    defp deps do
+    [
+      # ...
+      # add these lines -->
+      {:ash, "~> 2.5.11"},
+      {:ash_authentication, "~> 3.7.3"},
+      {:ash_authentication_phoenix, "~> 1.4.5"},
+      {:ash_postgres, "~> 1.3.2"},
+      {:elixir_sense, github: "elixir-lsp/elixir_sense", only: [:dev, :test]}
```
Comment (zachdaniel) [inline documentation/tutorials/getting-started-with-ash-authentication-phoenix.md:35]: This part should become unnecessary as soon as a new elixir-ls release happens. Been waiting a while on that, though.

**Comment by zachdaniel** [comment]
I think this is great, but the main question I have is wether this should replace this guide(there are some obvious improvements like the typo around `path_path), or if it should be used as a separate more "comprehensive" guide, since it touches on setting up `ash_postgres` and `ash_authentication` and `ash_authentication_phoenix`.

**Comment by wintermeyer** [comment]
My 2 cents about replacement or two versions: Keep it simple. One version only. One of the major problems in writing this tutorial was that there is a sea of documents in the Ash universe. Very bumpy to find the right stuff. Very fragmented. The core team knows this all but for new users it is not easy. 

A newbie can use it as a cooking recipe right away and a pro can jump to the parts he/she need.

**Comment by wintermeyer** [comment]
Sorry for the mix up. => https://github.com/team-alembic/ash_authentication_phoenix/pull/90

**Pull request closed without merging**